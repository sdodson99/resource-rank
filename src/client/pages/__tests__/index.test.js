import 'regenerator-runtime';
import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import Index from '../index.page';

describe('home page', () => {
  it('should render correctly', () => {
    const page = createRenderer().render(<Index />);

    expect(page).toMatchSnapshot();
  });
});
