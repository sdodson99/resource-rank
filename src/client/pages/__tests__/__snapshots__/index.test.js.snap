// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render correctly 1`] = `
<Layout>
  <NextSeo
    openGraph={
      Object {
        "title": "Home - Resource Rank",
      }
    }
    title="Home"
  />
  <Hero />
  <FeatureListing
    features={
      Array [
        Object {
          "description": "We rank resources based on feedback from users. This allows you to quickly find the most useful resources.",
          "imageSrc": "/img/instantly-find.svg",
          "title": "Instantly Find the Best Resources",
        },
        Object {
          "description": "Not sure where to start? Find your next learning journey by browsing through our list of topics.",
          "imageSrc": "/img/explore.svg",
          "title": "Explore Topics",
        },
        Object {
          "description": "Rate resources based on helpfulness for learning a topic.",
          "imageSrc": "/img/rate-resources.svg",
          "title": "Give Your Opinion",
        },
        Object {
          "description": "Add your own new topics and resources that are useful.",
          "imageSrc": "/img/add-your-own.svg",
          "title": "Contribute",
        },
      ]
    }
  />
  <CallToAction />
</Layout>
`;
